samples/distributed/polybench.py

```diff
@@ -0,0 +1,1293 @@
+# Copyright 2019-2022 ETH Zurich and the DaCe authors. All rights reserved.
+""" Assortment of distributed Polybench kernels. """
```

Comment:
does this qualify as a sample? this is quite long

Reply:
maybe a benchmark in a new folder called `benchmarks/`, but not a sample to me

Reply:
It is *many* samples, but it is better to group them together to work around `MPI_Finalize`.